app/application/usecases/auth/signin_usecase.py

```
@@ -0,0 +1,13 @@
+from app.application.services.auth import SignInService
+from app.interface.schemas.common.requests import SigninRequest
+
+
+class SignInUseCase:
+    def __init__(self, service: SignInService) -> None:
+        self._service = service
+
+    async def signin(self, request: SigninRequest):
```

Ruff (0.12.2)

Missing return type annotation for public function `signin`

(ANN201)